borg/archiver.py

```diff
@@ -1399,7 +1440,7 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='ARCHIVE',
                                type=location_validator(archive=True),
                                help='name of archive to create (must be also a valid directory name)')
-        subparser.add_argument('paths', metavar='PATH', nargs='+', type=str,
+        subparser.add_argument('paths', metavar='PATH', nargs='*', type=str,
```

Comment:
the use case for not having to specify any paths is if all roots are given e.g. in a patterns file, right?

so, would `borg create --patterns-from-file=patt.txt repo::arch /doesnotexist` be a workaround?
does it emit a warning about that then? is rc == 1 because of that?